### ISSUE TITLE
Update Apollo Examples to use 2.1 best practices

### ISSUE THREAD

**Issue by Aubron** [learnapollo/learnapollo#92]
2.1 emphasizes the use of their new Component based methodology, with things like the `<Query>` and `<Mutation>` component. They've pushed for it to the point that it is quickly becoming difficult to find documentation for the 2.0 methods used in learnapollo.

A full update to 2.1 for apollo sections seems to be in order.

**Comment by marktani** [comment]
Thanks a lot for your feedback, @Aubron.

Learn Apollo has been discontinued in favor of [How to GraphQL](https://www.howtographql.com/), could you make a similar feature request [here](https://github.com/howtographql/howtographql)?

That being said, if you feel like contributing an update to LA, I'd be happy to review it.

**Comment by Aubron** [comment]
Yeah of course. If nothing else I might make a PR with a warning on this section.

New issue: 
https://github.com/howtographql/howtographql/issues/570
